app/models/chart_data.rb

```diff
@@ -15,6 +17,15 @@ def data
     [values]
   end
 
+  def success?
+    ! data.first.series_data.nil?
+  rescue EnergySparksNotEnoughDataException
+    false
+  rescue => exception
+    Rails.logger.error "Chart generation failed unexpectedly for #{chart_type} and #{@school.name} - #{exception.message}"
```

Comment:
Should we Rollbar these?

Reply:
Ah yes, we should, especially with the extra check above.